### DISCUSSION TITLE
Possible workaround for people with flickering issues on Windows

### DISCUSSION THREAD

**Discussion by MateusAuri** [FreeTubeApp/FreeTube#4463] [General]
Running from Windows 11 with a GTX 750, latest nightly build (0.19.1 3739). Not posting this as an issue because I kinda solved it already.
I was having flickering issues when going fullscreen... tried messing with most available settings in the NVIDIA control panel to no avail. Then I saw someone mention disabling hardware acceleration on chrome helped them...

So I tried to go to Windows settings, on **System>Display>Graphics**, then I selected freetube.exe and set it to "**Power saving**" mode, for it to use the integrated graphics (Intel HD 4400) instead of my GTX 750, and it seems to have made the flickering stop altogether. So there's something to suggest whenever someone else gets the same problem in the future. Only thing is that now sometimes I get some screen tearing.

Moreso, if it helps with anything in tackling this problem from the dev side... in my case, the flickering would only happen when in fullscreen and when the player UI was moving. 
When playing a 60fps video, only the player UI flickered. 
30fps videos would flicker along with the UI. 
Seems to be some kind of mismatch between the video and the player animations messing with the fullscreen GPU optimizations I guess? Not sure how it works anyway :P also I had just updated to the nightly build because in stable 0.19.1, instead of flickering, the video would lag behind the audio for a second or so whenever the player UI animated over it, but thankfully it doesn't do that anymore.

**Comment by absidue** [comment]
Please check the NVIDIA settings mentioned here: https://github.com/FreeTubeApp/FreeTube/issues/2235#issuecomment-1133918031

**Comment by MateusAuri** [comment]
> Please check the NVIDIA settings mentioned here: [#2235 (comment)](https://github.com/FreeTubeApp/FreeTube/issues/2235#issuecomment-1133918031)

Yeah, I came across that, tried it and it didn't help at all in my case. That's why I made this post.

**Comment by absidue** [comment]
You can also try disabling features like GSYNC, as those are likely to not like the video and UI frame rate not matching up.
